eth_account/account.py

```diff
@@ -507,7 +508,7 @@ def sign_message(self, signable_message: SignableMessage, private_key):
         :param private_key: the key to sign the message with
         :type private_key: hex str, bytes, int or :class:`eth_keys.datatypes.PrivateKey`
         :returns: Various details about the signature - most importantly the fields: v, r, and s
-        :rtype: ~eth_account.datastructures.AttributeDict
+        :rtype: ~eth_account.datastructures.SignedMessage
```

Comment:
Interesting that the doctests (which were supposedly checked recently) didn't catch that the returned value changes below.

I would expect there to be a number of places in the docs where this would change.

Reply:
I just spent 1,5 hour to depuzzle this and here's the best I know so far:

1. We never had any doctest running in this repo so far. I wasn't able to find a single CI run that ran any doctests (even though they exist)

2. Then I found https://github.com/ethereum/eth-account/pull/94 which seems to fix this (it reports running 50 tests)

I'd happily rebase this on top of #94 but in case #94 needs more time to finish it should be easy to rebase that one on top of these changes, too.

Reply:
🤦 I totally lost track of #94. I just updated and it should be ready to merge. @cburgdorf I'm happy to pull your changes into #94, or vice versa, whatever is easier! Just let me know.

Reply:
@kclowes I don't have merge rights in this repo. If you have merge rights then feel free to merge both PRs in any preferred order.